optimum/exporters/neuron/config.py

```diff
@@ -61,3 +65,85 @@ class TextNeuronDecoderConfig(NeuronDecoderConfig):
     """
 
     pass
+
+
+class TextSeq2SeqNeuronConfig(NeuronConfig):
+    """
+    Handles encoder-decoder-based text architectures.
+    """
+
+    DUMMY_INPUT_GENERATOR_CLASSES = (
+        DummyTextInputGenerator,
+        DummySeq2SeqDecoderTextInputGenerator,
+        DummySeq2SeqPastKeyValuesGenerator,
+    )
+
+    @property
+    def is_decoder(self) -> bool:
+        raise NotImplementedError()
```

Comment:
Why?
Should we make this class an abstract class?

Reply:
euh, it's not necessary actually, will just remove it.